benchmark_test.go

```diff
@@ -609,3 +619,217 @@ func Benchmark_S3_segupload(b *testing.B) {
 	*/
 
 }
+
+func benchmarkBloom(strs [][]byte) {
+	numItems := uint(len(strs)) // num of items the filter will store
+	// false positive rate (BLOOM_COLL_PROBABILITY): 0.001
+	bloom := bloom.NewWithEstimates(numItems, utils.BLOOM_COLL_PROBABILITY)
+
+	for _, str := range strs {
+		bloom.Add(str)
+	}
+
+	fname := "bloomfilter.bin"
```

Comment:
Do we want to delete this file after benchmarking?

Same question for other files we write to

Reply:
addressed in the latest commit.